phys2bids/cli/run.py

```diff
@@ -57,49 +64,52 @@ def _get_parser():
     optional.add_argument('-sub', '--subject',
                           dest='sub',
                           type=str,
-                          help=('To be specified with \"-heur\". Code of '
-                                'subject to process.'
-                                'Specify path to it if necessary.'),
+                          help='Specify alongside \"-heur\". Code of '
+                                'subject to process.',
                           default=None)
     optional.add_argument('-ses', '--session',
                           dest='ses',
                           type=str,
-                          help=('To be specified with \"-heur\". Code of '
-                                'session to process.'
-                                'Specify path to it if necessary.'),
+                          help='Specify alongside \"-heur\". Code of '
+                                'session to process.',
                           default=None)
     optional.add_argument('-chtrig', '--channel-trigger',
                           dest='chtrig',
                           type=int,
-                          help=('The number corresponding to the trigger channel.'
-                                ' Channel numbering starts with 0'),
+                          help='The column number of the trigger channel. '
+                               'Channel numbering starts with 0. '
+                               'Default is 0.',
                           default=0)
     optional.add_argument('-chsel', '--channel-selection',
                           dest='chsel',
                           nargs='*',
                           type=int,
-                          help='The number corresponding to the channels to process.',
+                          help='The column numbers of  the channels to process. '
+                               'Default is to process all channels.',
                           default=None)
     optional.add_argument('-ntp', '--numtps',
                           dest='num_timepoints_expected',
                           type=int,
-                          help='Number of expected timepoints.',
+                          help='Number of expected timepoints (TRs). '
+                               'Default is 0.',
```

Comment:
```suggestion
                               'Default is 0., Which means the control of first trigger won't happen.',
```

Reply:
"Control of the first trigger" is not clear to me

Reply:
The guess of the start timing of the neuroimaging acquisition?

Reply:
Ah I see what you mean now.  What about: "Default is 0, which means the estimate of when the neuroimaging acquisition started will be wrong."

Reply:
It's not going to be wrong, but it's not going to be performed, hence your file starting time is 0.
`the estimate of when the neuroimaging acquisition started will not take place.`?

Reply:
Ah I seeeeee now! 
"Default 0. Note: the estimation of when the neuroimaging acquisition started cannot take place with this default"

Reply:
Can you change this in the file? After that, we can merge!

Reply:
Oh I see. Yep